lab/style-script/gss-functions.md

```diff
@@ -711,6 +711,58 @@ Outputs:
 
 - `number`
 
+### `Floor(value)`
+
+Returns the largest integer not greater than the input value.
```

Comment:
```suggestion
Returns the largest integer less than or equal to the input value.
```

Reply:
For me, it is more clear without negation. But, if you prefer _not greater_, you can leave it like that.

Reply:
Definitely agree, it is more clear without negation!